internal/db/db.go

```diff
@@ -212,6 +213,9 @@ func (db *DB) AddPolicy(
 	ctx context.Context,
 	policy string,
 ) (client.AddPolicyResult, error) {
+	ctx, span := tracer.Start(ctx, "AddPolicy")
```

Comment:
suggestion: I do not know about performance/C-GO, but we can probably avoid having to manually specify `"AddPolicy"` by using the runtime package and stuff like [runtime.CallersFrames](https://golang.org/pkg/runtime/#CallersFrames) instead.

Reply:
I can look into that.

Reply:
Update: I was able to get this working. Thanks for the suggestion!

Reply:
The preview above is outdated. Should be fixed in the actual code.